pkg/controller/control.go

```diff
@@ -1156,6 +1167,77 @@ func (c *Controller) GetLatestMetics() *ptypes.Metrics {
 	return metrics
 }
 
+func (c *Controller) ShowProfiler() (string, bool, error) {
```

Comment:
Can we include the profiler info in API `ControllerServer.VolumeGet` output?

Reply:
Sure, I could also add it to VolumeGet